### PR TITLE
Fix candidate, source handlers unowned comments bug

### PR DESCRIPTION
**Pull request by acrellin**
Closes https://github.com/skyportal/skyportal/issues/860 

This error was also occurring on the single candidate view page, and was fixed in the source (single, multiple) get handler as well.

### CODE REVIEW

File: skyportal/handlers/api/candidate.py
Comment (acrellin) [inline skyportal/handlers/api/candidate.py:279]: Note I'm only changing `Filter.query` -> `DBSession().query(Filter)` here

File: skyportal/handlers/api/candidate.py
Comment (dmitryduev) [inline skyportal/handlers/api/candidate.py:295]: Could you please add sorting in a reversed order here? Currently, I see:
image

File: skyportal/handlers/api/source.py
Comment (dmitryduev) [inline skyportal/handlers/api/source.py:228]: 👍

File: skyportal/handlers/api/source.py
Comment (dmitryduev) [inline skyportal/handlers/api/source.py:313]: Sort?

**Review by dmitryduev — APPROVED** [review]
Awesome, thanks for the quick fix! Just one minor comment from me.

**Comment by acrellin** [comment]
@dmitryduev Done (see updated diff)

**Review by dmitryduev — APPROVED** [review]
Great, will merge once Danny approves.

**Review by dannygoldstein — APPROVED** [review]

**Comment by dannygoldstein** [comment]
Sorry this took me so long

**Pull request merged**
